src/directory/directory.rs

```diff
@@ -43,7 +43,6 @@ impl RetryPolicy {
 }
 
 /// The `DirectoryLock` is an object that represents a file lock.
-/// See  [`LockType`](struct.LockType.html)
```

Comment:
`DirectoryLockGuard` ? It's private though.

Reply:
No clue what it was